cpp/Ice/secure/ClientOpenSSL.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) ZeroC, Inc.
+
+#include "Client.h"
+#include <Ice/Ice.h>
+
+using namespace std;
+
```

Comment:
I would add a general comment like:

```cpp
// The main function of the OpenSSL client (Linux).
```

Same for the other main functions. While I know OpenSSL corresponds to Linux, I find the platform mapping Schannel and SecureTransport less obvious.

Reply:
Fixed as suggested.